### PR TITLE
Implement blob sidecar signatures

### PR DESCRIPTION
**Pull request by henridf**

### CODE REVIEW

File: beacon_chain/spec/signatures.nim [hunk 1]
```diff
@@ -107,6 +107,15 @@ func get_block_signature*(
 
   blsSign(privkey, signing_root.data)
 
+# https://github.com/ethereum/consensus-specs/blob/v1.3.0/specs/deneb/validator.md#constructing-the-signedblobsidecars
```
Comment (tersec) [inline beacon_chain/spec/signatures.nim]:
```suggestion
# https://github.com/ethereum/consensus-specs/blob/v1.4.0-alpha.3/specs/deneb/validator.md#constructing-the-signedblobsidecars
```

`get_blob_sidecar_signature` is unchanged between `v1.3.0` and `v1.4.0-alpha.3`

File: beacon_chain/validators/validator_pool.nim [hunk 2]
```diff
@@ -623,6 +623,20 @@ proc getBlockSignature*(v: AttachedValidator, fork: Fork,
                 proofs)
       await v.signData(web3SignerRequest)
 
+# https://github.com/ethereum/consensus-specs/blob/v1.3.0/specs/deneb/validator.md#constructing-the-signedblobsidecars
```
Comment (tersec) [inline beacon_chain/validators/validator_pool.nim]:
```suggestion
# https://github.com/ethereum/consensus-specs/blob/v1.4.0-alpha.3/specs/deneb/validator.md#constructing-the-signedblobsidecars
```

`get_blob_sidecar_signature` is unchanged between `v1.3.0` and `v1.4.0-alpha.3`

**Review by tersec — APPROVED** [review]

**Comment by tersec** [comment]
@henridf not sure why the Jenkins CI didn't pick this up -- maybe try rebasing it to `unstable` or similar just to try to trigger that again

**Comment by henridf** [comment]
@tersec ok, done

**Comment by tersec** [comment]
Needs to rerun `make test` to rgenerate the test summaries on it, because the test summaries got out of sync

https://ci.status.im/blue/organizations/jenkins/nimbus-eth2%2Fplatforms%2Fmacos%2Faarch64/detail/PR-5072/2/pipeline shows the error message -- it's showing that the result of `make test` it got differed from what was in the commit

**Pull request merged**